homework/style.css

```diff
@@ -1,3 +1,112 @@
 .alert-error {
```

Comment:
Your application is unusable on a mobile phone. This is what it looks like.

![screenshot 2018-12-14 at 13 47 04](https://user-images.githubusercontent.com/2788771/50004240-d94a9d80-ffa6-11e8-9a59-5605e6ba6f57.png)